dist/selleckt-legacy.js

```diff
@@ -105,7 +105,7 @@ function MultiSelleckt(options){
 
 MultiSelleckt.prototype = Object.create(SingleSelleckt.prototype);
 
-MultiSelleckt.prototype._mutationHandler = function (mutations){
+MultiSelleckt.prototype._mutationHandler = function(mutations){
```

Comment:
I left the `dist` files as tests were failing without them as discussed with @stefanbuck